### DISCUSSION TITLE
Some questions about searches and accepted manifests

### DISCUSSION THREAD

**Discussion by Ra2-IFV** [ScoopInstaller/Scoop#6115] [Help and Questions] [answered]
## Search

how does `scoop search` work? It seems only searches app names and binaries.
I wish to install [Rivatuner Statistics Server](https://www.guru3d.com/download/rtss-rivatuner-statistics-server-download/), I ran `scoop search rivatuner` and returned no matches. so I plan to write a manifest for it. I decided to use Github Search to make a double check, and, yeah, scoop does have a existing manifest, but named in short: `rtss`.
Maybe we should also search the description? or add a new field called `keywords`?

## Manifests

I maintain a small bucket at [here](https://github.com/ifvlaboratory/ScoopBucket), I think some of them can be merged into official buckets: [Thorium](https://github.com/Alex313031/thorium) [Betterbird](https://www.betterbird.eu/) [ntttcp](https://github.com/microsoft/ntttcp), but not sure about this. Can someone check it?

**Comment by HUMORCE** [comment, marked as answer]
`scoop-search` searches name and binaries, [scoop.sh](https://scoop.sh/) searchs name and description.

---

> I maintain a small bucket at [here](https://github.com/ifvlaboratory/ScoopBucket), I think some of them can be merged into official buckets: [Thorium](https://github.com/Alex313031/thorium) [Betterbird](https://www.betterbird.eu/) [ntttcp](https://github.com/microsoft/ntttcp), but not sure about this. Can someone check it?

1. Search app name in [official buckets](https://github.com/ScoopInstaller/Scoop?tab=readme-ov-file#known-application-buckets).
2. [Criteria](https://github.com/ScoopInstaller/Scoop/wiki/Criteria-for-including-apps-in-the-main-bucket) checks
3. [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md)

`thorium` already exists in extras bucket.
users is requesting for `betterbird`: https://github.com/ScoopInstaller/Extras/issues/12415

**Reply by Ra2-IFV** [reply]
OK

**Reply by Ra2-IFV** [reply]
btw thorium has different versions for cpu (avx2 avx sse) so I wrote 3 manifests
